.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - '12.18.3'
+cache:
+  npm: true
```

Comment:
Just a heads up I heard that enabling cache for node_modules might cause issues: 

https://stackoverflow.com/questions/42521884/should-i-have-travis-cache-node-modules-or-home-npm

This issue might have been fixed, we'll just need to keep an eye out for it.

Otherwise, enabling caching is gonna be an awesome change.

Reply:
I also saw this when setting the cache to true, but I also read that you can easily purge the Travis cache in case some inconsistencies start showing up between Travis builds and local testing! Perhaps we can add a warning line above this.

Reply:
Ended up adding in a Troubleshooting part in the README regarding cache instead of an awkward comment